frontend/src/lib/dashboard/getQueryResults.ts

```diff
@@ -40,6 +40,10 @@ export async function GetMetricQueryRange(
 		throw new Error(error);
 	}
 
+	if (props.formatForWeb) {
```

Comment:
this is added to not process any data when we are having the `formatForWeb` flag to be true as all the processing has been done at server already